### DISCUSSION TITLE
Help users track their calories intake and weight

### DISCUSSION THREAD

**Discussion by unfazedEgnimadevspace** [unfazedEgnimadevspace/Gracker#38] [Ideas]
Hello @americodls 

Can we create a feature where we help users track their calories intake as well as their current weight  tp go along with their workout routines

What do you think about this idea.

Thank you 
Akalugo Daniel

**Comment by americodls** [comment]
Hey @unfazedEgnimadevspace 

It seems to be very useful as well.

Do you have any idea on how to start that?

**Comment by unfazedEgnimadevspace** [comment]
We can have them manually input them everyday(send them email reminders) or anytime they want workout they can input them before  and maybe after they start exercising . Then have a graph that displays calories taken each day and visually rises or drops depending on if the user's calorie intake was high or low on that particular day

**Comment by unfazedEgnimadevspace** [comment]
What do you think about the idea @americodls, should i create an issue them start implementing it code wise

**Reply by americodls** [reply]
Of course, go ahead!🚀
It's your project mate! 😁
I will review as soon as I can. 👍
